src/todoStatus.js

```diff
@@ -0,0 +1,106 @@
+import ToDo from './toDo.js';
+import DeleteButton from './delete-1.png';
+
+// Add items to UI
+export function populateList() {
+  const todoList = document.getElementById('todo-list');
+  todoList.innerHTML = '';
+
+  ToDo.list.forEach((item) => {
+    const listItem = document.createElement('li');
+    listItem.setAttribute('id', item.index);
+    listItem.classList = 'item-container';
+
+    listItem.innerHTML = `
+    <input class="checkbox" type="checkbox">
+    <span>${item.description}</span>
+    <textarea class="text-area" maxlength="30">${item.description}</textarea>
+    <img class="cancel-btn" src="${DeleteButton}">
+    `;
+
+    todoList.appendChild(listItem);
+
+    const checkbox = listItem.querySelector('input');
```

Comment:
I recommend you to place all declarations at the top to follow JavaScript best practices.